src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1290,7 +1290,7 @@ private static function processAttachmentUrls(array $urls): array
 			$filetype = strtolower(substr($mediatype, 0, strpos($mediatype, '/')));
 
 			if ($filetype == 'audio') {
-				$attachments[$filetype] = ['type' => $mediatype, 'url' => $href, 'height' => null, 'size' => null];
+				$attachments[] = ['type' => $mediatype, 'url' => $href, 'height' => null, 'size' => null, 'name' => ''];
```

Comment:
For audio in the previous code the `type` was set to "audio" and the `mediaType` was set to `$mediatype`.